Deprecation warning in jib-maven-plugin:test sources (assertThat)

solution is to use: `org.hamcrest.MatcherAssert.assertThat` instead of `org.junit.Assert.assertThat`
```
> Task :jib-maven-plugin:compileTestJava                                                                 
/home/appu/src/jib/jib-maven-plugin/src/test/java/com/google/cloud/tools/jib/maven/MavenSettingsServerCredentialsTest.java:58: warning: [deprecation] <T>assertThat(T,Matcher<? super T>) in Assert has been deprecated
      Assert.assertThat(
            ^
  where T is a type-variable:
    T extends Object declared in method <T>assertThat(T,Matcher<? super T>)
/home/appu/src/jib/jib-maven-plugin/src/test/java/com/google/cloud/tools/jib/maven/SkippedGoalVerifier.java:42: warning: [deprecation] <T>assertThat(T,Matcher<? super T>) in Assert has been deprecated
    Assert.assertThat(
          ^
  where T is a type-variable:
    T extends Object declared in method <T>assertThat(T,Matcher<? super T>)
/home/appu/src/jib/jib-maven-plugin/src/test/java/com/google/cloud/tools/jib/maven/SkippedGoalVerifier.java:61: warning: [deprecation] <T>assertThat(T,Matcher<? super T>) in Assert has been deprecated
    Assert.assertThat(
          ^
  where T is a type-variable:
    T extends Object declared in method <T>assertThat(T,Matcher<? super T>)
/home/appu/src/jib/jib-maven-plugin/src/test/java/com/google/cloud/tools/jib/maven/MavenProjectPropertiesExtensionTest.java:300: warning: [deprecation] <T>assertThat(T,Matcher<? super T>) in Assert has been deprecated
      Assert.assertThat(
            ^
  where T is a type-variable:
    T extends Object declared in method <T>assertThat(T,Matcher<? super T>)
/home/appu/src/jib/jib-maven-plugin/src/test/java/com/google/cloud/tools/jib/maven/skaffold/InitMojoTest.java:65: warning: [deprecation] <T>assertThat(T,Matcher<? super T>) in Assert has been deprecated
    Assert.assertThat(output, CoreMatchers.startsWith("BEGIN JIB JSON"));
          ^
  where T is a type-variable:
    T extends Object declared in method <T>assertThat(T,Matcher<? super T>)
/home/appu/src/jib/jib-maven-plugin/src/test/java/com/google/cloud/tools/jib/maven/MavenSettingsProxyProviderTest.java:277: warning: [deprecation] <T>assertThat(T,Matcher<? super T>) in Assert has been deprecated
      Assert.assertThat(
            ^
  where T is a type-variable:
    T extends Object declared in method <T>assertThat(T,Matcher<? super T>)
/home/appu/src/jib/jib-maven-plugin/src/test/java/com/google/cloud/tools/jib/maven/MavenSettingsProxyProviderTest.java:290: warning: [deprecation] <T>assertThat(T,Matcher<? super T>) in Assert has been deprecated
      Assert.assertThat(
            ^
  where T is a type-variable:
    T extends Object declared in method <T>assertThat(T,Matcher<? super T>)
7 warnings
```

--- a/jib-maven-plugin/src/test/java/com/google/cloud/tools/jib/maven/MavenProjectPropertiesExtensionTest.java
+++ b/jib-maven-plugin/src/test/java/com/google/cloud/tools/jib/maven/MavenProjectPropertiesExtensionTest.java
@@ -40,6 +40,7 @@ import org.apache.maven.plugin.descriptor.PluginDescriptor;
 import org.apache.maven.plugin.logging.Log;
 import org.apache.maven.project.MavenProject;
 import org.hamcrest.CoreMatchers;
+import org.hamcrest.MatcherAssert;
 import org.junit.Assert;
 import org.junit.Before;
 import org.junit.Test;
@@ -297,7 +298,7 @@ public class MavenProjectPropertiesExtensionTest {
       Assert.fail();
     } catch (JibPluginExtensionException ex) {
       Assert.assertEquals("invalid base image reference:  in*val+id", ex.getMessage());
-      Assert.assertThat(
+      MatcherAssert.assertThat(
           ex.getCause(), CoreMatchers.instanceOf(InvalidImageReferenceException.class));
     }
   }

--- a/jib-maven-plugin/src/test/java/com/google/cloud/tools/jib/maven/MavenSettingsProxyProviderTest.java
+++ b/jib-maven-plugin/src/test/java/com/google/cloud/tools/jib/maven/MavenSettingsProxyProviderTest.java
@@ -23,6 +23,7 @@ import org.apache.maven.settings.Proxy;
 import org.apache.maven.settings.Settings;
 import org.apache.maven.settings.crypto.SettingsDecrypter;
 import org.hamcrest.CoreMatchers;
+import org.hamcrest.MatcherAssert;
 import org.junit.Assert;
 import org.junit.Before;
 import org.junit.BeforeClass;
@@ -274,7 +275,7 @@ public class MavenSettingsProxyProviderTest {
           badProxyEncryptedSettings, settingsDecrypter);
       Assert.fail();
     } catch (MojoExecutionException ex) {
-      Assert.assertThat(
+      MatcherAssert.assertThat(
           ex.getMessage(),
           CoreMatchers.startsWith("Unable to decrypt proxy info from settings.xml:"));
     }
@@ -287,7 +288,7 @@ public class MavenSettingsProxyProviderTest {
           mixedProxyEncryptedSettings, emptySettingsDecrypter);
       Assert.fail();
     } catch (MojoExecutionException ex) {
-      Assert.assertThat(
+      MatcherAssert.assertThat(
           ex.getMessage(),
           CoreMatchers.startsWith("Unable to decrypt proxy info from settings.xml:"));
     }

--- a/jib-maven-plugin/src/test/java/com/google/cloud/tools/jib/maven/MavenSettingsServerCredentialsTest.java
+++ b/jib-maven-plugin/src/test/java/com/google/cloud/tools/jib/maven/MavenSettingsServerCredentialsTest.java
@@ -22,6 +22,7 @@ import java.nio.file.Path;
 import java.nio.file.Paths;
 import java.util.Optional;
 import org.hamcrest.CoreMatchers;
+import org.hamcrest.MatcherAssert;
 import org.junit.Assert;
 import org.junit.Before;
 import org.junit.Test;
@@ -55,7 +56,7 @@ public class MavenSettingsServerCredentialsTest {
       mavenSettingsServerCredentials.inferAuth("badServer");
       Assert.fail();
     } catch (InferredAuthException ex) {
-      Assert.assertThat(
+      MatcherAssert.assertThat(
           ex.getMessage(),
           CoreMatchers.startsWith("Unable to decrypt server(badServer) info from settings.xml:"));
     }

--- a/jib-maven-plugin/src/test/java/com/google/cloud/tools/jib/maven/SkippedGoalVerifier.java
+++ b/jib-maven-plugin/src/test/java/com/google/cloud/tools/jib/maven/SkippedGoalVerifier.java
@@ -24,7 +24,7 @@ import java.nio.file.Paths;
 import org.apache.maven.it.VerificationException;
 import org.apache.maven.it.Verifier;
 import org.hamcrest.CoreMatchers;
-import org.junit.Assert;
+import org.hamcrest.MatcherAssert;
 
 /** A simple verifier utility to test goal skipping across all our jib goals. */
 class SkippedGoalVerifier {
@@ -39,7 +39,7 @@ class SkippedGoalVerifier {
     verifier.executeGoal("jib:" + goal);
 
     Path logFile = Paths.get(verifier.getBasedir(), verifier.getLogFileName());
-    Assert.assertThat(
+    MatcherAssert.assertThat(
         new String(Files.readAllBytes(logFile), StandardCharsets.UTF_8),
         CoreMatchers.containsString(
             "[INFO] Skipping containerization because jib-maven-plugin: skip = true\n"
@@ -58,7 +58,7 @@ class SkippedGoalVerifier {
     verifier.executeGoal("jib:" + goal);
 
     Path logFile = Paths.get(verifier.getBasedir(), verifier.getLogFileName());
-    Assert.assertThat(
+    MatcherAssert.assertThat(
         new String(Files.readAllBytes(logFile), StandardCharsets.UTF_8),
         CoreMatchers.containsString(
             "[INFO] Skipping containerization of this module (not specified in jib.containerize)\n"

--- a/jib-maven-plugin/src/test/java/com/google/cloud/tools/jib/maven/skaffold/InitMojoTest.java
+++ b/jib-maven-plugin/src/test/java/com/google/cloud/tools/jib/maven/skaffold/InitMojoTest.java
@@ -30,6 +30,7 @@ import java.util.regex.Pattern;
 import org.apache.maven.it.VerificationException;
 import org.apache.maven.it.Verifier;
 import org.hamcrest.CoreMatchers;
+import org.hamcrest.MatcherAssert;
 import org.junit.Assert;
 import org.junit.ClassRule;
 import org.junit.Test;
@@ -62,7 +63,7 @@ public class InitMojoTest {
     verifier.verifyErrorFreeLog();
     Path logFile = Paths.get(verifier.getBasedir()).resolve(verifier.getLogFileName());
     String output = String.join("\n", Files.readAllLines(logFile, StandardCharsets.UTF_8)).trim();
-    Assert.assertThat(output, CoreMatchers.startsWith("BEGIN JIB JSON"));
+    MatcherAssert.assertThat(output, CoreMatchers.startsWith("BEGIN JIB JSON"));
 
     Pattern pattern = Pattern.compile("BEGIN JIB JSON\r?\n(\\{.*})");
     Matcher matcher = pattern.matcher(output);

